File: Actividad P1A5 Creacion de clases/FechaTest.py
# Ian Alexander Paz Hernández
# Carrera: TI
# 00603548

class Fecha:
    def __init__(self, mes=0, dia=0, anio=0):
        self.unMes = mes
        self.unDia = dia
        self.unAnio = anio
        
    def mostrarFecha(self):
        return f"{self.unMes}/{self.unDia}/{self.unAnio}"

# FechaTest
fechaActual = Fecha()

print("Ingrese los siguientes valores en numeros")
fechaActual.unDia = int(input("Que dia es hoy: "))
fechaActual.unMes = int(input("En que mes estamos: "))
fechaActual.unAnio = int(input("En que anio estamos: "))

print("\n---------------- FECHA ACTUAL ----------------")
print(fechaActual.mostrarFecha())

File: Actividad P1A5 Creacion de clases/FechaTest.java
// Ian Alexander Paz Hernández
// Carrera: TI
// 00603548

import java.util.Scanner;

class Fecha {
    public int unMes;
    public int unDia;
    public int unAnio;

    public Fecha() {
        unMes = 0;
        unDia = 0;
        unAnio = 0;
    }

    public Fecha(int mes, int dia, int anio) {
        unMes = mes;
        unDia = dia;
        unAnio = anio;
    }

    public String mostrarFecha() {
        return unMes + "/" + unDia + "/" + unAnio;
    }
}

public class FechaTest {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        Fecha fechaActual = new Fecha();
        
        System.out.println("Ingrese los siguientes valores en numeros");
        
        System.out.println("Que dia es hoy: ");
        fechaActual.unDia = input.nextInt();
        
        System.out.println("En que mes estamos: ");
        fechaActual.unMes = input.nextInt();
        
        System.out.println("En que anio estamos: ");
        fechaActual.unAnio = input.nextInt();
        
        System.out.println("\n---------------- FECHA ACTUAL ----------------");
        System.out.println(fechaActual.mostrarFecha());
    }
}